graphql-kotlin-schema-generator/src/main/kotlin/com/expediagroup/graphql/generator/generateSchema.kt

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2020 Expedia, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.expediagroup.graphql.generator
+
+import com.expediagroup.graphql.TopLevelObject
+import com.expediagroup.graphql.generator.types.generateMutations
+import com.expediagroup.graphql.generator.types.generateQueries
+import com.expediagroup.graphql.generator.types.generateSubscriptions
+import graphql.schema.GraphQLSchema
+
+/**
+ * Internal use only. Please use [com.expediagroup.graphql.toSchema] instead.
```

Comment:
whats the point of having this function? wouldn't it make more sense to just add it in to the `toSchema` instead?

Reply:
guess you kept it separate to allow usage from `federation` but i think it is still pretty confusing

Reply:
I have now made this the only publicly exposed top level function. We need to have this public so we can call it from the federation module. I also don't want to add it onto the generator class as that is now just the state of the current generating schema.